TestDriver/src/reduxElements.js

```diff
@@ -0,0 +1,57 @@
+import Heap from "react-native-heap-analytics";
+import { applyMiddleware, createStore } from "redux";
+
+// Action Types
+
+const INCREMENT = "INCREMENT";
+const DECREMENT = "DECREMENT";
+
+// Actions
+
+export const incrementAction = amount => {
```

Comment:
I don't know that we have a JS styleguide specifying export format, but our TS style guide specifies a different export format, and I think we should apply that to JS: https://paper.dropbox.com/doc/Capture-guide-to-writing-Typescript--AThgc8_LF5LMV7tnwm9LswHKAg-f2vfyHsDHDrE6PBdJ1MvF